api/src/main/resources/api.yaml

```diff
@@ -82,6 +82,139 @@ paths:
                 $ref: '#/components/schemas/GetUserDetailsResponseUser'
         '404':
           description: "If userId is invalid"
+  /houses:
+    get:
+      tags:
+        - house
```

Comment:
Let's change tag name to `Houses` and add it also in line 9 of this file to the tags list